docs/user/whats-new.asciidoc

```diff
@@ -64,38 +64,31 @@ image::images/currierlocation.png[Example of currier locations in relations to s
 
 [discrete]
 ==== Stacked bar chart legend
+The stacked bar chart legends are now arranged in descending order based on their values, with the segment having the highest total value shown at the top.
 
-The order of stacked bar chart legends is now based on popularity with the most important element displayed on top. You no longer need to scroll to the bottom of the legend to find the most popular data in the chart. 
-
-Previously:
-[role="screenshot"]
-image::images/oldwayhighlighted.png[Example of the old ordering of the chart legend, width=80%]
-
-New in 8.9:
 [role="screenshot"]
-image::images/newwayhighlighted.png[Example of the new ordering of the chart legend, width=80%]
+image::images/newwayhighlighted.png[Example of the new ordering of the chart legend, width=90%]
 
 [discrete]
 ==== Step curve
-
-**Lens** now supports three new options for line charts. The three options are straight, smooth, and step. Step curve line charts help users see clearly when data changes at irregular intervals, which helps to avoid misinterpretation. When creating visualizations select **Line** in the visualization type dropdown. Then use the visual options menu to choose between straight, smooth, or step in the Line interpolation dropdown. 
+**Lens** now supports a new line interpolation option for line charts. Step curve line charts help users see clearly when data changes at irregular intervals, reducing misinterpretation. When creating visualizations select **Line** in the visualization type dropdown. Then use the **Visual options** menu to choose step in the **Line interpolation** dropdown.
 
 [role="screenshot"]
 image::images/stepcurve.gif[An short video displaying the three new chart options]
 
 [discrete]
 ==== Significance (significant terms)
 
-This new ranking method in the top values breakdown displays data by significance (significant terms). It permits us to get the terms that are important and likely the most critical for a given query. It’s a powerful way to find anomalies in your data with multiple use cases. Read more {ref}/search-aggregations-bucket-significantterms-aggregation.html[here].
-This is not just based on the most popular terms in a set. When calculating significant terms, Elasticsearch looks for significant changes in popularity of a specific term between two sets of data. A foreground set (the data returned by a query) and a background set (the data in our indices). This is supported on all charts. 
+This new ranking method in the top values breakdown displays data by significance (significant terms). It helps you identify the key and most crucial terms for a specific query. It’s a powerful way to find anomalies in your data with multiple use cases. Read more {ref}/search-aggregations-bucket-significantterms-aggregation.html[here].
```

Comment:
Using "here" for a link is not good for accessibility.  Instead say something specific like, "Refer to the xxx docs".